cmake/stm32/utilities.cmake

```diff
@@ -94,62 +106,55 @@ function(stm32_fetch_cube)
             continue()
         endif()
         
-		FetchContent_GetProperties(${CUBE_NAME} POPULATED CUBE_POPULATED)
-        if(NOT CUBE_POPULATED)
-            set(FETCHCONTENT_QUIET FALSE) # To see progress
-            FetchContent_Populate(${CUBE_NAME})
-        endif()
-        
+		FetchContent_MakeAvailable(${CUBE_NAME})

```

Comment:
Original comment from @Inujel 
I changed 
```
FetchContent_GetProperties(${CMSIS_NAME_L} POPULATED CMSIS_POPULATED)
if(NOT CMSIS_POPULATED)
		set(FETCHCONTENT_QUIET FALSE) # To see progress
		FetchContent_Populate(${CMSIS_NAME})
endif()
```
and similar to
```
FetchContent_MakeAvailable(${CMSIS_NAME})
```

Because
1. It's advised by the [CMake doc](https://cmake.org/cmake/help/latest/module/FetchContent.html)
2. There was a bug when `stm32_fetch_cube` was called multiple times from different scopes. After the first call:
	- `STM32_CUBE_${FAMILY}_PATH` is not set because we're in another scope (so the `continue()` path is not taken)
	- `FetchContent_Populate` is not called a second time, so `${CUBE_NAME_L}_SOURCE_DIR` is not set
	- `STM32_CUBE_${FAMILY}_PATH` is set to an empty string in the parent scope

Using `FetchContent_MakeAvailable` always sets `${CUBE_NAME_L}_SOURCE_DIR`

Reply:
You seem to have worked on the subject. I do not see any drawback, just ensure the example are still building.
It's good work 👍